test/test_loc_handler.erl

```diff
@@ -0,0 +1,16 @@
+-module(test_loc_handler).
+
+-export([init/2, content/0]).
+
+init(Req0 = #{path_info := [<<"location">> | _]}, State) ->
+    ct:pal("Receiving request in test_loc_handler: ~p~n", [Req0]),
+    Resp = cowboy_req:reply(200,
+	#{<<"content-type">> => <<"application/json">>},
+	test_loc_handler:content(),
```

Comment:
Why here is used call like `test_loc_handler:content()`? Is there is some issue if you will call just `content()`?

Reply:
It is due to the way meck is used to influence the results. The handler will not update its response if the unqualified call is used.